lua/wire/client/text_editor/texteditor.lua

```diff
@@ -2325,6 +2357,16 @@ function EDITOR:AC_Check( notimer )
 			self:AC_SetVisible( false )
 			return
 		end
+		
+		if suggestions[1].str( suggestions[1] ) == "vec" or suggestions[1].str( suggestions[1] ) == "vec4" then
+			for k=1, table.Count(PickColorFuncs) do
```

Comment:
Never use `table.Count`. It's a function added by garry and it's complete crap. `table.Count` loops through the entire table to get the amount of elements. Use `#t` instead. This operator gets the size instantly. The reason garry added `table.Count` is because `#t` only works on numerical sequential tables (yours is one such table so it's fine), while garry's function works on any table, even if it has holes in the indexes, or if it uses strings or vectors or players as indexes. This is still stupid though because if you are using such a table, 99.9999% of the time, the actual size of it doesn't matter (you just use a for each loop to iterate it). And the times when the size does matter, you simply keep a seperate variable of the size somewhere.

Be careful when using garry's functions (they all start with a capital letter) because many of them are crap and promote bad code.